File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int count=0;
        for(int i=0;i<nums.length-1;i++){
            if(nums[i]!=nums[i+1]){
                nums[count]=nums[i];
                count++;
            }




        }

        nums[count]=nums[(nums.length)-1];
        int k=count+1;
       
        count++;
        while(count<nums.length){
            nums[count]=0;
            count++;
        }
        return k;

    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int prev=0;
        for(int i=1;i<nums.size();i++){
            if(nums[i]!=nums[prev]){
                swap(nums[i],nums[prev+1]);
                prev++;

            }

        }

        return prev+1;
    }
    void swap(int &a,int &b){
        int temp=a;
        a=b;
        b=temp;
    }
};